astro/src/content/docs/apis/_lambda-type.astro

```diff
@@ -15,7 +15,7 @@ const lambdas: LambdaDoc[] = (lambdasEntry.data as LambdaDoc[])
     .sort((a, b) => a.typeText.localeCompare(b.typeText));
 
 const { prefix, showRequired, showSince } = Astro.props;
-const name = `${prefix || ''}type`;
+const name = `${prefix || ''}.type`;
```

Comment:
Looks like this breaks some of the callers. I think the caller is expected to add the `.`. 

With this change, I see `search..type`, `.type`, `lamba.type` on the Lambda API page.

Reply:
This was kinda messed up, should be fixed now.